src/pages/users/Windows/UsersWindow.js

```diff
@@ -38,12 +40,15 @@ const UsersWindow = ({ labels, maxAccess, recordId }) => {
         <DocTypeTab labels={labels} maxAccess={maxAccess} storeRecordId={storeRecordId} />
       </CustomTabPanel>
       <CustomTabPanel index={4} value={activeTab}>
-        <RowAccessTab labels={labels} maxAccess={maxAccess} storeRecordId={storeRecordId} />
+        <SitesTab labels={labels} maxAccess={maxAccess} recordId={storeRecordId} />
```

Comment:
you can send only storeRecordId,

Reply:
how? i'm renaming it to recordId and using it in children as recordId